pkg/sql/schemachanger/scbuild/table.go

```diff
@@ -132,7 +194,7 @@ func (b *buildContext) alterTableAddColumn(
 		familyID = b.findOrAddColumnFamily(
 			table, familyName, d.Family.Create, d.Family.IfNotExists,
 		)
-	} else {
+	} else if !d.IsVirtual() { // FIXME: Compute columns should not have families?
```

Comment:
Stored ones definitely need families. Virtual ones I would think can live without them but I don't know whether or not they do.